src/Authorization/Models/ContextRequest.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Altinn.Platform.Authorization.Models
+{
+    /// <summary>
+    /// This model describes an authorization event. An authorization event is an action triggered when a user requests access to an operation
+    /// </summary>
+    public class ContextRequest
```

Comment:
Summary describes this model as: "This model describes an authorization event" then should the model not be called something to this effect? ContextRequest to me sounds like a input request model of some sort.

Reply:
Oh, it's a copy/paste bug it seems

Reply:
I still think both the ContextRequest and AccessSubject etc model naming here should have a stronger separation from the actual XacmlContextRequest etc.
And perhaps have their own namespace

Reply:
Actually this model is not in use. My last commit failed to remove this. The implementation now is that the enriched XacmlContextRequest is serialized as string and stored as jsonb object in database.